### DISCUSSION TITLE
1.6版本部署文档中给出的 halo.service 模板链接失效

### DISCUSSION THREAD

**Discussion by KatanaNeko** [halo-dev/halo#7041] [General]
[作为服务运行](https://v1.legacy-docs.halo.run/getting-started/install/linux#%E4%BD%9C%E4%B8%BA%E6%9C%8D%E5%8A%A1%E8%BF%90%E8%A1%8C)部分，第二步给出的代码中的[Halo 官方的 halo.service 模板链接](https://dl.halo.run/config/halo.service)失效，以及第四步给出的代码是否是完整的halo.service 代码？
![图片](https://github.com/user-attachments/assets/4f9b4f89-cca6-43b8-a36b-770a53181972)

**Comment by ruibaby** [comment]
可以参考：https://docs.halo.run/getting-started/install/jar-file#%E4%BD%9C%E4%B8%BA%E6%9C%8D%E5%8A%A1%E8%BF%90%E8%A1%8C

此外，[1.x 已经停止维护](https://www.halo.run/archives/halo-1.x-eol)，不再推荐使用。

**Comment by KatanaNeko** [comment]
好的，感谢
